controller/cmd/service-mirror/events_formatting.go

```diff
@@ -65,23 +65,23 @@ func (gtm gatewayMetadata) String() string {
 
 // Events for cluster watcher
 func (rsc RemoteServiceCreated) String() string {
-	return fmt.Sprintf("RemoteServiceCreated: {service: %s, gatewayData: %s}", formatService(rsc.service), rsc.gatewayData)
+	return fmt.Sprintf("TargetServiceCreated: {service: %s, gatewayData: %s}", formatService(rsc.service), rsc.gatewayData)
```

Comment:
I think "Remote" is actually appropriate here because the perspective is clear.  This is in the logs of the service mirror controller running in the source cluster, so it's clear that remote services are services running in the target cluster.  

Alternatively, "TargetClusterServiceCreated" could work.  To put it more generally, "source" and "target" are adjectives we use for clusters.

Reply:
Good point. Since this is relative to the source cluster, I kept local/remote terminology.